shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/rql/DataSourceQueryResultSet.java

```diff
@@ -81,6 +81,7 @@ public boolean next() {
         result.put("maxPoolSize", dataSourceParameter.getMaxPoolSize());
         result.put("minPoolSize", dataSourceParameter.getMinPoolSize());
         result.put("readOnly", dataSourceParameter.isReadOnly());
+        result.put("customPoolProps", dataSourceParameter.getCustomPoolProps());
```

Comment:
The result of dataSourceParameter.getCustomPoolProps() may be NULL or EMPTY, so a check is needed.

Reply:
> The result of dataSourceParameter.getCustomPoolProps() may be NULL or EMPTY, so a check is needed.

Has been updated, please review, thank you.

Reply:
It's OK now.